app/views/projects/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1><%= @project.name %></h1>
+<p>Material: <%= @project.material %></p>
+<h3>Challenge theme: <%= @project.challenge.theme %> </h3>
+<h3>Number of Contestants: <%=@project.contestants.count %></h3>
```

Comment:
This works, but doing `.count` is performing a calculation, and as such has a better home in a model method. That way, we can also test it before we show the user a calculation that isn't tested. This would be considered an MVC infraction.